frostsnapp/native/src/api.rs

```diff
@@ -2,21 +2,29 @@ use crate::device_list::DeviceList;
 pub use crate::ffi_serial_port::{
     PortBytesToReadSender, PortOpenSender, PortReadSender, PortWriteSender,
 };
+use crate::sink_wrap::SinkWrap;
 pub use crate::FfiCoordinator;
 pub use crate::{FfiQrEncoder, FfiQrReader, QrDecoderStatus};
 use anyhow::{anyhow, Context, Result};
 pub use bitcoin::psbt::Psbt as BitcoinPsbt;
 pub use bitcoin::Network as RBitcoinNetwork;
 pub use bitcoin::Transaction as RTransaction;
+use bitcoin::{network, NetworkKind};
 use flutter_rust_bridge::{frb, RustOpaque, StreamSink, SyncReturn};
+use frostsnap_coordinator::bitcoin::chain_sync::{
+    default_electrum_server, ElectrumConnection, SUPPORTED_NETWORKS,
+};
 pub use frostsnap_coordinator::bitcoin::wallet::ConfirmationTime;
-pub use frostsnap_coordinator::bitcoin::{chain_sync::ChainSync, wallet::FrostsnapWallet};
+pub use frostsnap_coordinator::bitcoin::{
+    chain_sync::{ChainClient, ChainStatus, ChainStatusState},
+    wallet::FrostsnapWallet,
+};
 pub use frostsnap_coordinator::firmware_upgrade::FirmwareUpgradeConfirmState;
 pub use frostsnap_coordinator::frostsnap_core;
 use frostsnap_coordinator::frostsnap_core::schnorr_fun::fun::hash::HashAdd;
 pub use frostsnap_coordinator::{
-    check_share::CheckShareState, keygen::KeyGenState, signing::SigningState, DeviceChange,
-    PortDesc,
+    check_share::CheckShareState, keygen::KeyGenState, persist::Persisted, signing::SigningState,
+    DeviceChange, PortDesc, Settings as RSettings,
```

Comment:
Why `RSettings`, `FSCoordinatorSettings`?

Reply:
R stands for Rust. We could make it a convention to prefix it with Fs instead.